hudi-common/src/main/java/org/apache/hudi/common/util/SerializationUtils.java

```diff
@@ -121,50 +116,16 @@ Object deserialize(byte[] objectData) {
 
     public Kryo newKryo() {
 
-      Kryo kryo = new KryoBase();
+      Kryo kryo = new Kryo();
       // ensure that kryo doesn't fail if classes are not registered with kryo.
       kryo.setRegistrationRequired(false);
       // This would be used for object initialization if nothing else works out.
-      kryo.setInstantiatorStrategy(new org.objenesis.strategy.StdInstantiatorStrategy());
+      kryo.setInstantiatorStrategy(new Kryo.DefaultInstantiatorStrategy(new StdInstantiatorStrategy()));
       // Handle cases where we may have an odd classloader setup like with libjars
       // for hadoop
       kryo.setClassLoader(Thread.currentThread().getContextClassLoader());
       return kryo;
     }
 
-    private static class KryoBase extends Kryo {
-      @Override
-      protected Serializer newDefaultSerializer(Class type) {
```

Comment:
The default is `FieldSerializer` and the description of setIgnoreSyntheticFields is `Controls if synthetic fields are serialized. Default is true.` So I think this Override might be unnecessary.

Reply:
👍